HIP/hip-991.md

```diff
@@ -105,13 +105,15 @@ We propose adding a fixed fee mechanism to the Hedera Consensus Service (HCS) fo
 * If the `allowCustomFeesPayment` flag is not set or is not true, the SDK returns an error.
 * The `allowCustomFeesPayment` flag exists only in the SDK and isn't enforced by the network.
 
-* **Allowance Transactions**: 
-   - An account submits a transaction to grant an allowance (total fees and maximum fee per message) to a topic for both HBAR and HTS tokens. This transaction type reuses the existing allowance concept, with no need for a separate delete allowance transaction. To remove an allowance, the user will submit the same transaction with the allowance values set to 0 or a dedicated flag indicating allowance removal.
-   - This allowance only permits the topic to pay for the custom fees when messages are submitted, not for any other purpose.
+#### Allowance Transactions
+
+* An account submits a transaction to grant an allowance (total fees and maximum fee per message) to a topic for both HBAR and HTS tokens. This transaction type reuses the existing allowance concept, with no need for a separate delete allowance transaction. To remove an allowance, the user will submit the same transaction with the allowance values set to 0 or a dedicated flag indicating allowance removal.
```

Comment:
I don't see a flag specified in the FeeScheduleAllowance messages below, so I'm thinking that we decided against a dedicated flag

Reply:
Good catch, thanks! I forgot to remove that statement.